### PR TITLE
[Web][Bug][SIMS #980] Changes to display banners based on hidden field values

### PR DESCRIPTION
**Pull request by dheepak-aot**
- [x] Added program details to initData for AEST client to let the validations happen and banner to display on view only mode
- [x] Modified form.io to use logic based on clientType to achieve read only view with hidden component values calculation.

### CODE REVIEW

**Review by henryoforeh-aot — APPROVED** [review]

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/web/src/views/institution/locations/programs/LocationProgramOffering.vue [hunk 1]
```diff
@@ -155,15 +154,31 @@ export default {
...
+         * The property clientType is populated for institution because
+         * the form.io for education program offering has a logic at it's root level panel
+         * to disable all the form inputs when clientType is not institution.
+         * The above mentioned logic is added to the panel of the form to display the
+         * form as read-only for ministry(AEST) user and also allow the hidden component values
+         * to be calculated.
+         *! If a form.io is loaded with readOnly attribute set to true, then the restricts
+         *! hidden components to calculate it's value by design.
+         */
+        initialData.value.clientType = ClientIdType.Institution;
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/locations/programs/LocationProgramOffering.vue:167]: Can we just assign it the `AuthService.shared.authClientType` directly and have it always assigned independent of the client to keep it consistent?

**Review by andrewsignori-aot — COMMENTED** [review]
Just one comment, please take a look.

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by ann-aot — APPROVED** [review]
👍

**Pull request merged**